packages/bot/src/views/commit.pug

```diff
@@ -0,0 +1,42 @@
+extends layout.pug
+block main
+  .container.mt-2
+    .row
+      .col-12
+        h3 #{commit.repo_name}@#{commit.id.slice(0, 8)}
+        p.lead "#{commit.message}"
```

Comment:
Can you help changes all commit messages display to use `pre` blocks without quoting? Commit messages are supposed to be pre-formatted (e.g. wrap at column 80) at commit time so using `pre` block is more appropiate.